components/camel-sjms/src/test/java/org/apache/camel/component/sjms/consumer/AsyncConsumerFalseTest.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.camel.component.sjms.consumer;
 
+
```

Comment:
Same here, could you please remove the extra lines that make the PR harder to review